lib/logger/test/logger/translator_test.exs

```diff
@@ -107,601 +108,619 @@ defmodule Logger.TranslatorTest do
-      Application.stop(:eex)
-    end) =~ """
-    Application eex exited: :stopped
-    """
+             :ok = Application.start(:eex)
+             Application.stop(:eex)
+           end) =~ """
+           Application eex exited: :stopped
+           """
   end
 
   test "translates application start" do
     assert capture_log(fn ->
-      Application.start(:eex)
-      Application.stop(:eex)
-    end) =~ """
-    Application eex started at #{inspect(node())}
-    """
+             Application.start(:eex)
+             Application.stop(:eex)
+           end) =~ """
+           Application eex started at #{inspect(node())}
+           """
   end
 
   test "translates Process crashes" do
     assert capture_log(:info, fn ->
-      {_, ref} = spawn_monitor(fn() -> raise "oops" end)
-      receive do: ({:DOWN, ^ref, _, _, _} -> :ok)
-      # Even though the monitor has been received the emulator may not have
-      # sent the message to the error logger
-      Process.sleep(200)
-    end) =~ ~r"""
-    \[error\] Process #PID<\d+\.\d+\.\d+>\ raised an exception
-    \*\* \(RuntimeError\) oops
-    """
+             {_, ref} = spawn_monitor(fn -> raise "oops" end)
+             receive do: ({:DOWN, ^ref, _, _, _} -> :ok)
+             # Even though the monitor has been received the emulator may not have
+             # sent the message to the error logger
+             Process.sleep(200)
+           end) =~ ~r"""
+           \[error\] Process #PID<\d+\.\d+\.\d+>\ raised an exception
+           \*\* \(RuntimeError\) oops
+           """
   end
 
   test "translates :proc_lib crashes" do
     {:ok, pid} = Task.start_link(__MODULE__, :task, [self()])
 
     assert capture_log(:info, fn ->
-      ref = Process.monitor(pid)
+             ref = Process.monitor(pid)
 
-      send(pid, :go)
-      receive do: ({:DOWN, ^ref, _, _, _} -> :ok)
-    end) =~ ~r"""
-    \[error\] Process #PID<\d+\.\d+\.\d+> terminating
-    \*\* \(exit\) an exception was raised:
-        \*\* \(RuntimeError\) oops
-    .*
-    Initial Call: Logger.TranslatorTest.task/1
-    Ancestors: \[#PID<\d+\.\d+\.\d+>\]
-    """s
+             send(pid, :go)
+             receive do: ({:DOWN, ^ref, _, _, _} -> :ok)
+           end) =~ ~r"""
+           \[error\] Process #PID<\d+\.\d+\.\d+> terminating
+           \*\* \(exit\) an exception was raised:
+               \*\* \(RuntimeError\) oops
+           .*
+           Initial Call: Logger.TranslatorTest.task/1
+           Ancestors: \[#PID<\d+\.\d+\.\d+>\]
+           """s
   end
 
   test "translates :proc_lib crashes with name" do
-    {:ok, pid} = Task.start_link(__MODULE__, :task,
-                                 [self(), fn() ->
-                                            Process.register(self(), __MODULE__)
-                                            raise "oops"
-                                          end])
+    {:ok, pid} =
+      Task.start_link(__MODULE__, :task, [
+        self(),
+        fn ->
```

Comment:
Please extract this function out to a variable. :)

Reply:
I see what you mean. the Task.start_link is a bit harder to read than necessary.